### PR TITLE
[AIG]Added autolog description

### PR DESCRIPTION
**Pull request by daisyfaithauma**
### Summary

- Added autolog description

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

File: src/content/docs/ai-gateway/reference/limits.mdx [hunk 1]
```diff
@@ -21,11 +21,12 @@ The following limits apply to gateway configurations, logs, and related features
 
 <sup>1</sup> If you have reached 10 million logs stored per gateway, new logs
 will stop being saved. To continue saving logs, you must delete older logs in
-that gateway to free up space or create a new gateway.
+that gateway to free up space or create a new gateway. Refer to [Auto Log Cleanup](/ai-gateway/observability/logging/) for
```
Comment (kathayl) [inline src/content/docs/ai-gateway/reference/limits.mdx]: for this link, is it possible to link directly to the auto log cleanup section on the page? 
#auto-log-cleanup

File: src/content/docs/ai-gateway/reference/limits.mdx [hunk 2]
```diff
@@ -21,11 +21,12 @@ The following limits apply to gateway configurations, logs, and related features
...
+more details on how to automatically delete logs.
 
-<sup>2</sup> If you have reached 100,000 logs stored per account, accross all
+<sup>2</sup> If you have reached 100,000 logs stored per account, across all
 gateways, new logs will stop being saved. To continue saving logs, you must
-delete older logs.
+delete older logs.  Refer to [Auto Log Cleanup](/ai-gateway/observability/logging/) for more details on how to automatically delete logs.
```
Comment (kathayl) [inline src/content/docs/ai-gateway/reference/limits.mdx]: same q as above

**Review by kathayl — APPROVED** [review]
1) since i'd updated the limits page to remove the repetitive "maximum" words, will this PR have a problem merging?
2) do we use italics elsewhere? is that the best practice for like feature names?

**Comment by daisyfaithauma** [comment]
> 1. since i'd updated the limits page to remove the repetitive "maximum" words, will this PR have a problem merging?
> 2. do we use italics elsewhere? is that the best practice for like feature names?

1. we will try and see haha
2. We use Italics this way https://developers.cloudflare.com/style-guide/formatting/code-conventions-and-format/#italics so I don't think its necessary here

**Review by kathayl — APPROVED** [review]

**Pull request merged**